docs/docs/1.7.x/security/api-server-auth.md

```diff
@@ -356,19 +350,21 @@ All users that provides client certificate are authenticated as user with name `
    :::
    ::::
 
-3. Configure `kumactl` with valid client certificates
+3. Configure `kumactl` with valid client certificate
    ```sh
    kumactl config control-planes add \
      --name=<NAME>
      --address=https://<KUMA_CP_DNS_NAME>:5682 \
      --client-cert-file=/tmp/tls.crt \
      --client-key-file=/tmp/tls.key \
-     --ca-cert-file=/tmp/ca.crt # CA cert used in "Encrypted communication" section
+     --ca-cert-file=/tmp/ca.crt
    ```
 
+   If you want to skip CP verification, use `--skip-verify` instead of `--ca-cert-file`.
+
 ## Multizone
 
-In multizone setup, the majority of actions are executed on the global control plane.
-However, some actions like generating dataplane tokens are available on zone control planes.
-Authentication credentials are not propagated from global control plane to zone control planes.
-Consistent user tokens across the whole setup can be achieved by manually synchronizing signing key from global to zone control planes. 
+In a multizone setup, users execute a majority of actions on the global control plane.
+However, some actions like generating dataplane tokens are available on the zone control plane.
+The global control plane doesn't propagate authentication credentials to the zone control plane.
+You can set up consistent user tokens across the whole setup can by manually synchronizing signing key from global to zone control planes. 
```

Comment:
Not a scope of this PR but how do I manually sync that up?

Reply:
changed synchronizing to copying. Just use kumactl/kubectl to copy global secret from Global CP and put it in Zone CP